### PR TITLE
fix(ide-extension): crash without git origin

### PR DESCRIPTION
**Pull request by openscript**
Uses `groupKey` "without-git", when there is no git origin

### CODE REVIEW

File: source-code/ide-extension/src/services/telemetry/implementation.ts [hunk 1]
```diff
@@ -60,14 +60,19 @@ async function capture(args: CaptureEventArguments) {
...
+			dir: await raw.findRoot({
+				fs,
+				filepath: workspaceRoot ?? process.cwd(),
+			}),
+		})
+		const gitOrigin = parseOrigin({ remotes })
+		return gitOrigin
+	} catch (e) {
+		console.warn(e)
+		return 'without-git'
```
Comment (samuelstroschein) [inline source-code/ide-extension/src/services/telemetry/implementation.ts:76]: return `undefined`

File: source-code/ide-extension/src/main.ts [hunk 2]
```diff
@@ -31,18 +31,14 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
...
-			console.warn(error)
-		}
+		const gitOrigin = await getGitOrigin()
+		telemetry.groupIdentify({
+			groupType: "repository",
+			groupKey: gitOrigin,
+			properties: {
+				name: gitOrigin,
+			},
+		})
```
Comment (samuelstroschein) [inline source-code/ide-extension/src/main.ts:41]: don't group identify if the origin is `undefined`. 

```ts

if (gitOrigin) {
telemetry.groupIdentify({
			groupType: "repository",
			groupKey: gitOrigin,
			properties: {
				name: gitOrigin,
			},
		})
}

File: source-code/ide-extension/src/services/telemetry/implementation.ts [hunk 3]
```diff
@@ -60,14 +60,19 @@ async function capture(args: CaptureEventArguments) {
...
-	return gitOrigin
+			dir: await raw.findRoot({
+				fs,
+				filepath: workspaceRoot ?? process.cwd(),
+			}),
+		})
+		const gitOrigin = parseOrigin({ remotes })
+		return gitOrigin
+	} catch (e) {
+		console.warn(e)
```
Comment (samuelstroschein) [inline source-code/ide-extension/src/services/telemetry/implementation.ts:75]: I would remove the console.warn. There is nothing "bad" going on if a directory has not git repo.

**Review by samuelstroschein — CHANGES_REQUESTED** [review]
Seems like you went the complicated route which would lead to bad telemetry data. 

### Request 

Simply don't perform a group identify if the origin is undefined, and don't use a stub ("without-git").

**Review by felixhaeberle — APPROVED** [review]
lgtm 👍

**Pull request merged**